resources/views/components/menu/account.blade.php

```diff
@@ -67,6 +67,9 @@
             @if($user->Permissions >= Permissions::Developer)
                 <x-dropdown-item :link="url('gameList.php?t=develop&f=2')">Want to Develop Games</x-dropdown-item>
             @endif
+            @if($user->ContribCount > 0)
+                <x-dropdown-item :link="route('user.dev-feed', $user->username)">Feed</x-dropdown-item>
```

Comment:
"Developer Feed" to prevent ambiguity in the future?
Or is it omitted here because of the dropdown header? 
There will be another activity feed based on player activities.

Not sure if we should shorten `dev` everywhere, maybe `user.developer-feed`?

Reply:
We're actually looking for more feedback on this type of decision [here](https://github.com/RetroAchievements/RAWeb/pull/1998#discussion_r1386652416) and [here](https://github.com/RetroAchievements/RAWeb/pull/1998#discussion_r1387278096).

Reply:
> "Developer Feed" to prevent ambiguity in the future?
> Or is it omitted here because of the dropdown header?

I feel it's consistent with "Tickets", "Sets", and "Claims", which all go to lists specific to the logged in developer.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/e41ad221-f1dc-47f8-8906-39335c170fce)

Reply:
Similar to the comment on th other PR - the route name could be `developer.feed` here

Reply:
> I feel it's consistent with "Tickets", "Sets", and "Claims", which all go to lists specific to the logged in developer.

Fair enough. We might still run into said ambiguity as soon as the other feed is added but that will take some time - we can revisit that later.

Reply:
> Similar to the comment on th other PR - the route name could be developer.feed here

Route name updated to `developer.feed` in https://github.com/RetroAchievements/RAWeb/pull/2004/commits/477f94bcfce505caee8c92018f794d7bf190569a.